front-end/src/UserHomePage.js

```diff
@@ -0,0 +1,121 @@
+  const year = today.getFullYear();
+  const date = today. getDate();
+  const currentDate = year + "/" + month + "/" + date;
+  console.log(currentDate);
+
+  return (
+    <React.Fragment>
+    <TopNavigation
+      identity={{
+        href: "#",
+        title: "Internal Survey Tool",
+        logo: {
+          src:"https://cdn4.vectorstock.com/i/1000x1000/88/53/document-with-stamp-and-pen-business-documents-vector-31048853.jpg",
+        alt: "Service"
+        }
+      }}
+      utilities={[
+        {
+          type: "button",
+          iconName: "notification",
+          title: "Notifications",
+          ariaLabel: "Notifications (unread)",
+          badge: true,
+          disableUtilityCollapse: false
+        },
+        {
+          type: "menu-dropdown",
+          text: "Hemanth Potti",
+          description: "email@example.com",
+          iconName: "user-profile",
+          items: [
+            { id: "profile", text: "Profile" },
+            { id: "preferences", text: "Preferences" },
+            { id: "security", text: "Security" },
+            {
+              id: "support-group",
+              text: "Support",
+              items: [
+                {
+                  id: "documentation",
+                  text: "Documentation",
+                  href: "#",
+                  external: true,
+                  externalIconAriaLabel:
+                    " (opens in new tab)"
+                },
+                { id: "support", text: "Support" },
+                {
+                  id: "feedback",
+                  text: "Feedback",
+                  href: "#",
+                  external: true,
+                  externalIconAriaLabel:
+                    " (opens in new tab)"
+                }
+              ]
+            },
+            { id: "signout", text: "Sign out" }
+          ]
+        }
+      ]}
+    />
+    
+    <ContentLayout
+      disableOverlap
+      header={
+        <SpaceBetween size="m" direction="horizontal">
+        <div className="parent">
+          <div className="child">
+            <div className="popup-center-horizontal">
+              <Header
+                description={false}
+                hasFrame={false}
+                headerBaseCounter={false}
+                headerBaseLinkVisible={false}
+                type="page-dark-header"> Welcome back Hemanth! 
+              </Header>;
+            </div>
+          </div>
+        </div>
+        </SpaceBetween>
+      }
+    >
+    </ContentLayout>
+    <div>
+      {data.map(list=>(
+        <div key={list.id}>
+          <Container header={<Header variant="h2" actions={
+              <SpaceBetween direction="horizontal" size="xs">
+                <Button variant="primary">Start</Button>
+              </SpaceBetween>}>{list.title}</Header>}> {list.due}    <Badge color="red">Overdue</Badge>
+              </Container>
+        </div>
+      ))}
+    </div>
+
+
+    </React.Fragment>
+  );
+}
+
+
+
```

Comment:
Just leave one line at the end of each file.

Reply:
Will keep that in mind